specification/attributes/column_naming_convention.md

```diff
@@ -30,6 +30,7 @@ Naming convention for columns appearing in billing data.
 ## Exceptions
 
 * Identifiers will use the "Id" abbreviation since this is a standard pattern across the industry.
+* Stock Keep Units will use the "SKU" abbreviation since this is a standard pattern across the industry.
```

Comment:
Have we closed on whether we'll use "SKU" vs. "Sku" in the column ID?  We should be explicit here.

Reply:
We have not.  Unfortunately I didn't realize this was a topic of debate -- otherwise I would have made a point of putting it on the agenda of yesterday's VS3 call.

While I had initially assumed "SKU", it seems that "Sku" makes more sense alongside the existing "Id".  There may be more acronyms that come down the pike, but for 1.0 it feels right to me to consistently apply Pascal case to acronyms.